build_r.R

```diff
@@ -88,9 +117,9 @@ version <- gsub(
 )
 tarball <- file.path(getwd(), sprintf("lightgbm_%s.tar.gz", version))
 
-cmd <- sprintf("R CMD INSTALL %s --no-multiarch --with-keep.source", tarball)
 if (INSTALL_AFTER_BUILD) {
-  .run_shell_command(cmd)
+  .run_shell_command("R", c("CMD", "INSTALL", tarball, "--no-multiarch", "--with-keep.source"))
 } else {
+  cmd <- sprintf("R CMD INSTALL %s --no-multiarch --with-keep.source", tarball)
```

Comment:
I believe we should save `c("CMD", "INSTALL", tarball, "--no-multiarch", "--with-keep.source")` in a variable and reuse it in these two commands.

Reply:
makes sense! I can change that

Reply:
updated in https://github.com/microsoft/LightGBM/pull/2965/commits/313827e90be65feafafac26023f75e5308e4e1c7